<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>928</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>701</width>
      <height>471</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../logo_exquisita.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#ff0099;&quot;&gt;BIENVENIDO AL MENU DE LA EXQUISITA&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEjercicios">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="mnuEmpleados"/>
    <addaction name="mnuUsuarios"/>
    <addaction name="mnuVentas"/>
   </widget>
   <addaction name="menuEjercicios"/>
  </widget>
  <action name="mnuEmpleados">
   <property name="text">
    <string>Empleados</string>
   </property>
  </action>
  <action name="mnuVentas">
   <property name="text">
    <string>Ventas</string>
   </property>
  </action>
  <action name="mnuUsuarios">
   <property name="text">
    <string>Usuarios</string>
   </property>
  </action>
  <action name="mnuAdministracion">
   <property name="text">
    <string>Administraci√≥n</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="imagenpizza.qrc"/>
 </resources>
 <connections/>
</ui>
